Core/src/org/sleuthkit/autopsy/modules/plaso/PlasoIngestModule.java

```diff
@@ -279,37 +278,41 @@ private void createPlasoArtifacts(String plasoDb, DataSourceIngestModuleProgress
                         MessageNotifyUtil.Message.info(Bundle.PlasoIngestModule_create_artifacts_cancelled());
                         return;
                     }
-                    
+
                     // lots of bad dates
                     if (resultSet.getString("sourcetype").equals("PE Import Time")) {
                         continue;
-                    }
-                    // bad dates and duplicates with what we have.
+                    } // bad dates and duplicates with what we have.
                     // TODO: merge results somehow
                     else if (resultSet.getString("source").equals("WEBHIST")) {
                         continue;
                     }
-                    
+
                     statusHelper.progress(resultSet.getString("filename"));
-                    
+
                     Content resolvedFile = getAbstractFile(resultSet.getString("filename"));
                     if (resolvedFile == null) {
                         logger.log(Level.INFO, "File from Plaso output not found.  Associating with data source instead: {0}", resultSet.getString("filename"));
                         resolvedFile = image;
                     }
-                    
+
                     try {
                         Collection<BlackboardAttribute> bbattributes = new ArrayList<>();
-                        bbattributes.add(new BlackboardAttribute(ATTRIBUTE_TYPE.TSK_DATETIME, Bundle.PlasoIngestModule_event_datetime(), resultSet.getLong("epoch_date")));
-                        bbattributes.add(new BlackboardAttribute(ATTRIBUTE_TYPE.TSK_DESCRIPTION, Bundle.PlasoIngestModule_event_description(), resultSet.getString("description")));
+                        bbattributes.add(new BlackboardAttribute(ATTRIBUTE_TYPE.TSK_DATETIME, MODULE_NAME, resultSet.getLong("epoch_date")));
+                        bbattributes.add(new BlackboardAttribute(ATTRIBUTE_TYPE.TSK_DESCRIPTION, MODULE_NAME, resultSet.getString("description")));
```

Comment:
Is MODULE_NAME a suitable replacement for the strings here?

Reply:
I am not sure.  Almost all modules put the module name here.  The old values were "Event Date Time" and "Event Description" which I didn't think were very informative.